rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -4,12 +4,11 @@
 import sys
 import time
 
+sys.path.insert(1, os.path.dirname(__file__)) # noqa: E261
 from deoplete_jedi import cache, profiler, utils, worker
```

Comment:
> What about a relative import instead?!

I thought I added a change to [deoplete that auto-injects the source directory](https://github.com/Shougo/deoplete.nvim/blob/106ad23a93486541f1a6da30f6ed81e699779685/rplugin/python3/deoplete/util.py#L90-L93)?